components/cam/src/physics/rrtmgp/cam_optics.F90

```diff
@@ -2,125 +2,75 @@ module cam_optics
 
    use shr_kind_mod, only: r8=>shr_kind_r8
    use assertions, only: assert, assert_valid, assert_range
-   use radiation_state, only: nlev_rad, ktop, kbot
-   use radiation_utils, only: handle_error
    use radconstants, only: nswbands, nlwbands
+   use rad_constituents, only: icecldoptics, liqcldoptics
+   use ebert_curry, only: ec_ice_optics_sw, ec_ice_optics_lw
+   use slingo, only: slingo_liq_optics_sw, slingo_liq_optics_lw
```

Comment:
It would be a bit more consistent to move these use statements down into the subroutines where they are used, like what was done for the Mitchell and GammaDist schemes. Not all that important though.